clusterregistry/selectors.yaml

```diff
@@ -1,17 +1,8 @@
 kind: ClusterSelector
 apiVersion: configmanagement.gke.io/v1
 metadata:
-  name: istio
+  name: asm
 spec:
   selector:
     matchLabels:
-      environment: istio
----
-kind: ClusterSelector
-apiVersion: configmanagement.gke.io/v1
-metadata:
-  name: csm
-spec:
-  selector:
-    matchLabels:
-      environment: csm
\ No newline at end of file
+      environment: asm
```

Comment:
Just a personal question. If we do not need to reference a ClusterSelector in other configs file, do we still need the cluster selector?

Reply:
technically for the current lab setup we don't need one. I just put it there as a reference / example.